workload-metrics/README.md

```diff
@@ -0,0 +1,9 @@
+# Workload Metrics example
+
+This tutorial demonstrates how to automatically scale your Google Kubernetes Engine (GKE)
```

Comment:
Hello Gari, I'm a bit confused about how this is different from the existing Custom Metrics Autoscaling tutorial and sample. 

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/master/custom-metrics-autoscaling 

https://cloud.google.com/kubernetes-engine/docs/tutorials/autoscaling-metrics

Reply:
I updated the text here as well, but basically, GKE has a new feature called "GKE workload metrics" which allows you to configure a policy to scrape Prometheus metrics from pods and send them to Cloud Monitoring.  So they are no longer "custom metrics" per se but rather "workload metrics" with their own Cloud Monitoring prefix.

We still need to deploy the adapter for HPA to work, but no longer need the prom-sd sidecar or to use sd metrics in the app.